### ISSUE TITLE
Support 2021 Blue Banners

### ISSUE THREAD

**Issue by fangeugene** [the-blue-alliance/the-blue-alliance#3943]
3 GDC winners got a blue banner. Need to investigate which other winners received blue banners for the at home challenges.

**Comment by ZachOrr** [comment]
Yeah! I looked in to this a while back - from https://www.firstinspires.org/robotics/frc/blog/2021-scores-deadline-award-shows-trophies-interviews

> And the Chairman’s Award winners, the Woodie Flowers Finalist Award winners, the three overall Game Design Challenge Winners, and the INFINITE RECHARGE at Home Skills Challenge winner in each group will receive the traditional blue banners!

We only awarded banners for the [skills competition winners](https://github.com/the-blue-alliance/the-blue-alliance/blob/95991a6af9f014cf4b5993240a74a0073aa6b90e/src/backend/common/consts/award_type.py#L115), so we should just have to support the GDC winners

**Comment by fangeugene** [comment]
It looks like the 3 overall GDC winners aren't even listed in the API. The designer's award is the one that's given to every group.

**Comment by fangeugene** [comment]
`https://frc-api.firstinspires.org/v2.0/2021/awards/gamce` is returning `Request Did Not Match Any Current API Pattern : Request Did Not Match Any Current API Pattern`

Looks like something changed with the API, and now `teamNumber` is a required field for getting all awards at an event.
https://frc-api-docs.firstinspires.org/?version=ce5fcb39-f244-4e6c-b1c9-74205ac45eb9

**Comment by BrennanB** [comment]
Tangentially related, 2021 District banners for district level and champs level is also something that currently isn't showing accurately it seems. 2020 seems correct, but 2021 is not

**Comment by fangeugene** [comment]
@BrennanB can you give a specific example?

**Comment by BrennanB** [comment]
Just realized I completely forgot to mention the context of the banners. Chairman's banners are the issue here. Particularly the teams that won at district level should have 2, currently they have 1 (so for Ontario 772/4476)

**Comment by ZachOrr** [comment]
Chairman's banners issue is being tracked at https://github.com/the-blue-alliance/the-blue-alliance/issues/3614
